roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -127,3 +135,9 @@
     dest: /etc/systemd/system/crio.service.d/http-proxy.conf
   notify: restart crio
   when: http_proxy is defined or https_proxy is defined
+
+- name: Install cri-o service
+  service:
+    name: crio
+    enabled: yes
+    state: started
```

Comment:
this is the reason why your job is failing: this piece of code was removed by PR #4607, just remove this task (see `restart crio` handler, needs a daemon reload first)

Reply:
I've re-added at in this commit, to test things, earlier commit failed to to crio service not being enabled [see this](https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/520295212). I think that the handler is missing the `enabled: yes` part. I'm trying that in the latest commit

Reply:
ouch, true..the handler needs the `enabled: yes` then..bug introduced by #4607